packages/renderer/src/lib/preferences/PreferencesKubernetesContextsRendering.svelte

```diff
@@ -74,8 +75,17 @@ async function handleDeleteContext(contextName: string) {
       aria-label="No Resource Panel"
       icon={EngineIcon}
       title="No Kubernetes contexts found"
-      message="Check that $HOME/.kube/config exists or KUBECONFIG environment variable has been set correctly."
-      hidden={$kubernetesContexts.length > 0} />
+      message="Check that Kubernetes context is created and selected. This can be done in Resources page."
+      hidden={$kubernetesContexts.length > 0}>
+      <Button
```

Comment:
I think it would be appropriate to have a button to open the folder where the kubeconfig is located. I like that in the current version we are explicitely saying that we are using the `$HOME/.kube/config`

Reply:
@axel7083 what about showing the kubecofig path + the button to change it? (something like in the settings->preferences->kubernetes)?

Reply:
I am fine with showing the kubeconfig path

Reply:
How does it look? Maybe add some new lines? 
![image](https://github.com/user-attachments/assets/f93ef78e-5852-4d7e-aaad-fc74b9405b16)

Reply:
:+1: